docs/rules/fixer-return.md

```diff
@@ -0,0 +1,41 @@
+# Enforces always return from a fixer function (fixer-return)
+
+In a fixable rule, missing return from a fixer function will not apply fixes.
+
+## Rule Details
+
+This rule enforces always return from a fixer function.
```

Comment:
Nitpick: I think it would be better to word this as "This rule enforces that fixer functions always return a value."